src/Stratis.Bitcoin.Features.Miner/PosMinting.cs

```diff
@@ -50,28 +48,36 @@ namespace Stratis.Bitcoin.Features.Miner
     /// </remarks>
     public class PosMinting
     {
-        public class StakeOutput
-        {
-            public UtxoStakeDescription UtxoStakeDescription;
-            public int Depth;
-        }
-
+        /// <summary>
+        /// Information related to UTXO that is required for staking.
+        /// </summary>
         public class UtxoStakeDescription
         {
-            public uint256 HashBlock;
-            public TxOut TxOut;
-            public OutPoint OutPoint;
-            public int OutputIndex;
-            public HdAddress Address;
-            public UnspentOutputs UtxoSet;
-            public WalletSecret Secret;
-            public Key Key;
+            /// <summary>Block's hash.</summary>
+            public uint256 HashBlock { get; set; }
```

Comment:
i thought i've mentioned spaces between properties already here, but it was probably somewhere else